meta-mender-qemu/recipes-core/systemd/files/01-logs-to-console.conf

```diff
@@ -0,0 +1,2 @@
+[Journal]
+ForwardToConsole=true
```

Comment:
I recall that we talked about this before: I think this is too verbose to include by default; this will also affect our demo. If this is something we could add to the tests only, it would be great. Maybe during test setup?